[ZT] Update Test DNS page (#6014)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/filtering/dns-policies/test-dns-filtering.md

    ![Verify that a domain is blocked when the block page is disabled.](/cloudflare-one/static/documentation/faq/blocked-disabled.png)

    If the [block page](/cloudflare-one/policies/filtering/configuring-block-page/) is enabled for the policy, you should see `NOERROR` in the answer section and `162.159.36.12` and `162.159.46.12` as the answers:

    ![Verify that a domain is blocked when the block page is disabled.](/cloudflare-one/static/documentation/faq/blocked-enabled.png)

### Test a security or content category

If you are blocking a [security category](/cloudflare-one/policies/filtering/dns-policies/#security-categories) or a [content category](/cloudflare-one/policies/filtering/dns-policies/#content-categories), you can test that the policy is working by using the [test domain](#common-test-domains) associated with each category.

Once you have configured your Gateway policy to block the category, the test domain will show a block page when you attempt to visit the domain in your browser, or will return `REFUSED` when you perform `dig` using the command-line interface.

#### Test domain format

> - **One-word category** — For categories with one-word names (for example, _Malware_), the test domain is of the form:

Reviewer: ```suggestion
- **One-word category** — For categories with one-word names (for example, _Malware_), the test domain uses the following format:
```

    ```txt
    <NAME_OF_CATEGORY>.testcategory.com
    ```

- **Multi-word category** — For categories with multiple words in the name (for example, _Parked & For Sale Domains_), the test domain uses the following format:

    - Remove any spaces between the words
    - Replace `&` with `and`

>     - All letters are lowercase

Reviewer: ```suggestion
    - Lowercase all letters
```

Adds parallelism in list
